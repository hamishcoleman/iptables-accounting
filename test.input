# Generated by iptables-save v1.8.7 on Thu Jan 01 10:00:00 1970
*filter
:INPUT DROP [10:100]
:FORWARD DROP [0:0]
:OUTPUT ACCEPT [20:200]
[11:110] -A INPUT -i lo -m comment --comment "Always Allow localhost" -j ACCEPT
*raw
:PREROUTING ACCEPT [30:300]
:OUTPUT ACCEPT [40:400]
[500:5000] -A PREROUTING -p tcp -m tcp --dport 22 -m comment --comment ACCT
[600:6000] -A PREROUTING -p udp -m udp --dport 53 -m comment --comment ACCT
[700:7000] -A OUTPUT -p tcp -m tcp --sport 22 -m comment --comment ACCT
[800:8000] -A OUTPUT -p udp -m udp --sport 53 -m comment --comment ACCT
COMMIT
# Completed on Thu Jan 01 10:00:00 1970
