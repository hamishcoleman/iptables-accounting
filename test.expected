# TYPE iptables_acct_packets_total counter
# TYPE iptables_acct_bytes_total counter
iptables_acct_packets_total{chain="PREROUTING",proto="tcp",port="22"} 500
iptables_acct_bytes_total{chain="PREROUTING",proto="tcp",port="22"} 5000
iptables_acct_packets_total{chain="PREROUTING",proto="udp",port="53"} 600
iptables_acct_bytes_total{chain="PREROUTING",proto="udp",port="53"} 6000
iptables_acct_packets_total{chain="OUTPUT",proto="tcp",port="22"} 700
iptables_acct_bytes_total{chain="OUTPUT",proto="tcp",port="22"} 7000
iptables_acct_packets_total{chain="OUTPUT",proto="udp",port="53"} 800
iptables_acct_bytes_total{chain="OUTPUT",proto="udp",port="53"} 8000
iptables_read_lines 15
buffer_capacity_bytes 1000
buffer_used_bytes 706
buffer_timestamp 1644144574
